packages/genesys-spark-chart-components/src/style/style.scss

```diff
@@ -2,4 +2,4 @@
  * This file combines all the exported library styles into a single stylesheet for consumption by applications.
  */
 
-@use '~genesys-spark/src/style/global.scss';
+@use '~genesys-spark/dist/scss/global.scss';
```

Comment:
This is the most frequent change in the PR. I added scss files to genesys-spark's distribution so they are available via node_modules and not via the cross monorepo file imports.